internal/confmapprovider/discovery/bundle/bundle.d/receivers/mongodb.discovery.yaml

```diff
@@ -42,10 +39,26 @@ mongodb:
               `--set splunk.discovery.receivers.mongodb.config.password="<password>"` or
               `SPLUNK_DISCOVERY_RECEIVERS_mongodb_CONFIG_username="<username>"` and
               `SPLUNK_DISCOVERY_RECEIVERS_mongodb_CONFIG_password="<password>"` environment variables.
+      - status: partial
+        regexp: 'SSL handshake received but server is started without SSL support'
+        log_record:
+          append_pattern: true
+          body: >-
+            Make sure the target database has SSL enabled or set insecure using the
+            `--set splunk.discovery.receivers.mongodb.config.tls::insecure="<boolean>"` command or the
+            `SPLUNK_DISCOVERY_RECEIVERS_mongodb_CONFIG_tls_x3a__x3a_insecure="<boolean>"` environment variable.
       - status: partial
         regexp: '.* failed to fetch index stats metrics: (Unauthorized) not authorized on admin to execute command .*'
         log_record:
           append_pattern: true
           body: >-
             Make sure the account used to access Mongodb has been given a clusterMonitor role in order to collect metrics.
             `db.grantRolesToUser('someUser', [{ role: 'clusterMonitor', db: 'admin' }])`
+      - status: partial
+        regexp: '.* tls: failed to verify certificate: .*'
```

Comment:
This for situations where mongodb is serving a certificate which can't be verified by the client (the receiver), right? The guidance for this should be for the user to ensure that the receiver has access to the CA file used to sign the server's certificate or they can skip certificate verification with the skip_verify option.

Reply:
Yes. Mongodb certificate is not found by receiver or not set.

Reply:
I will push a commit with the guidance changed, let me know your thoughts.